packages/studio-ui/.size-limit.cjs

```diff
@@ -1,6 +1,6 @@
 module.exports = [
   {
-    path: "lib/src/index.js",
+    path: "lib/**/*.js",
```

Comment:
more of a question for me, but what is the reason behind this change?

Reply:
Adding a dynamic import to the bundle adds additional js files to studio-ui/lib that don't live under the src folder